packages/plugins/graphql/server/src/services/builders/type.ts

```diff
@@ -272,12 +328,39 @@ export default (context: Context) => {
 
       return objectType({
         name,
-
         definition(t) {
+          if (
```

Comment:
Could you add a small comment with why we're doing this. Might bne helpful for future contributors